coops/sentinels.tex

```diff
@@ -7,138 +7,127 @@
 
 \textbf{Source:} \href{https://discord.com/channels/740870068178649108/1233112440322002964/1233112440322002964}{Archon Studio Discord}
 
+\subsection*{\MakeUppercase{The Story}}
+
+\textit{There is a world beyond this world, and it is filled with beasts and monsters, barely held captive by a formation of four sacred Obelisks.
+  But the protection of the stones is weakening, and a breach is imminent.
+  When it happens, only you and your ally stand between the hordes and the total devastation of these lands.
+}
+
 \subsection*{\MakeUppercase{Scenario length}}
 
-This scenario plays out over 12 rounds.
+This scenario lasts for 12 rounds.
```

Comment:
I'd prefer "play over" over "last". Maybe "This scenario is played over 12 rounds"?